tests/framework/installer/ceph_installer.go

```diff
@@ -119,6 +113,23 @@ func (h *CephInstaller) CreateCephOperator(namespace string) (err error) {
 	return nil
 }
 
+func (h *CephInstaller) startAdmissionControllers(namespace string) error {
+	err := h.k8shelper.CreateNamespace(namespace)
+	if err != nil {
+		if errors.IsAlreadyExists(err) {
+			logger.Warningf("Namespace %q already exists!!!", namespace)
+		} else {
+			return fmt.Errorf("failed to create namespace %q. %v", namespace, err)
+		}
+	}
+	h.k8shelper.MakeKubeConfig()
+	err = h.k8shelper.MakeContext().Executor.ExecuteCommandWithEnv([]string{fmt.Sprintf("NAMESPACE=%s", namespace)}, "bash", "-c", "tests/scripts/deploy_test.sh")
```

Comment:
My local tests also failed because the current working directory was github.com/rook/rook/tests/integration instead of github.com/rook/rook as these tests assume. We'll need this to work in both environments.

Reply:
I have added findRookRoot func. It should now be able to find the scripts if the cwd is inside rook repo.